OpenOfficeIntegration/Renderer/Renderer.py

```diff
@@ -898,21 +898,26 @@ def insertQuote(self, text):
       self.insert_paragraph_character(avoid_empty_paragraph=True)
       self.changeParaStyle(oldStyle)
 
-   def insertInlineQuote(self, text):
+   def insertInlineQuote(self, content):
       if self.needSpace():
          self.insertString(" ")
 
       if self._currentLanguage == 'de':
          self.insertString('„')
       else: 
          self.insertString('"')
-      self.render(text)
-      self.smartSpace()
+
+      # Do not put a space in front of quoted content.
+      self._lastItem = None
+      self.render(content)
+
       if self._currentLanguage == 'de':
          self.insertString('“')
       else:
          self.insertString('"')
 
+      self.smartSpace()
```

Comment:
Moved the application of the smart space hook to the end of the function as it is going to effect the next element and not the quotation mark.